local-workflow.js

```diff
@@ -0,0 +1,24 @@
+/********************************************************************************
+ * Copyright (c) 2025 Precies. Software OU and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+// @ts-check
+const publishExtensionsScript = require("./scripts/publish-extensions");
+const buildExtensionScript = require("./scripts/build-extension");
+const publishExtensionScript = require("./scripts/publish-extension");
+
+(async () => {
+    process.env.EXTENSIONS = "dfinity-foundation.vscode-motoko";
+    process.env.SKIP_PUBLISH = "false";
+    process.env.FORCE = "true";
```

Comment:
I'd prefer to only set these if they're not set by the executing context. Also, let's default to not settings extensions and since it's the local workflow for debugging, skip publishing by default.